Sources/POI/OAPOIType.h

```diff
@@ -46,6 +46,7 @@
 - (void)setAdditional:(OAPOIBaseType *)parentType;
 
 - (UIImage *)mapIcon;
+- (UIImage *)iconWithoutParent;
```

Comment:
No such field in Android

Reply:
Android also has no such field "- (UIImage *) icon"
"iconWithoutParent" is short for "icon" and is required for correct display so as not to pull unnecessary icons